src/test/python/python_interface.py

```diff
@@ -0,0 +1,7 @@
+import jnius_config
+jnius_config.set_classpath("target/anserini-0.1.1-SNAPSHOT-fatjar.jar")
```

Comment:
Is there a way to do this without hardcoding the version number? We may need to manually bump this if we hardcode it.

Reply:
One way I can think of is to use `os` to find a jar in `target` matching some regex and get the path of the jar.